src/MobileUI/Pages/LeaderboardPage.xaml

```diff
@@ -20,38 +20,37 @@
         <mct:StatusBarBehavior StatusBarColor="{StaticResource Background}"
           StatusBarStyle="LightContent" />
     </ContentPage.Behaviors>
-    <Grid>
+    <Grid RowDefinitions="Auto,*">
+        <controls:SegmentedControl
+            Grid.Row="0"
+            Segments="{Binding Periods}"
+            HeightRequest="50"
+            Padding="15,10,15,0"
+            SelectedSegment="{Binding SelectedPeriod, Mode=OneWayToSource}">
+            <controls:SegmentedControl.Behaviors>
+                <mct:EventToCommandBehavior
+                    EventName="SelectionChanged"
+                    Command="{Binding FilterByPeriodCommand}" />
+            </controls:SegmentedControl.Behaviors>
+        </controls:SegmentedControl>
         <RefreshView
+            Grid.Row="1"
             Command="{Binding RefreshLeaderboardCommand}"
             IsRefreshing="{Binding IsRefreshing}">
             <CollectionView
                 ItemsSource="{Binding SearchResults}"
                 x:Name="LeadersCollection"
-                ItemsUpdatingScrollMode="KeepItemsInView"
-                ItemSizingStrategy="MeasureFirstItem"
```

Comment:
Can we please leave `ItemSizingStrategy="MeasureFirstItem"`? The default value is `MeasureAllItems` which will degrade CollectionView performance. All items in the list are the same, so there is no need to measure them all.

Reply:
Ah, thanks, I didn't mean to commit this so this slipped through. Reverted now.